src/client/components/RoomSelector.jsx

```diff
@@ -39,7 +39,7 @@ const RoomSelector = ({socket, joinRoom}) => {
       <h1>Public rooms</h1>
       <p>(click to join)</p>
       <div className="room-list">
-        {rooms.map(room => <div key={room.id} onClick={room.id !== currentRoomId ? (() => {
+        {rooms.map((room, i) => <div key={room.id} tabIndex={i + 1} onClick={room.id !== currentRoomId ? (() => {
```

Comment:
any reason to not just use `tabIndex=0` ?

Reply:
With tabindex=0 you have to press `Shift` `Tab`, not just `Tab` and you will be traversing list starting from last room.